storage/database.go

```diff
@@ -24,7 +24,7 @@ func (s *dbRepoStore) Create(repo *model.Repository) error {
 }
 
 func (s *dbRepoStore) Get(id kallax.ULID) (*model.Repository, error) {
-	q := model.NewRepositoryQuery().FindByID(id)
+	q := model.NewRepositoryQuery().WithReferences(nil).FindByID(id)
```

Comment:
Why is needed call explicitly `WithReferences` to nil value?

Reply:
Because we're not passing any condition to filter the references, we want them all